src/spectrum-exporter.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @typedef {object} ExportOptions
+ * @property {string} columnDelimiter
+ * @property {string} stringDelimiter
+ * @property {boolean} quoteStrings
+ */
+
+/**
+ * @constructor
+ * @param {object} fftOutput
+ * @param {ExportOptions} [opts={}]
+ */
+export function SpectrumExporter(fftData, opts = {}) {
+  var opts = _.merge(
```

Comment:
`opts` is a function parameter already
```suggestion
  opts = _.merge(
```

Reply:
The all SonarCloud issues are resolved